src/main/java/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,21 @@
+package service.impl;
+
+import db.Storage;
+import java.util.Map;
+import service.ReportGenerator;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public String getReport() {
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append("fruit,quantity").append(System.lineSeparator());
+
+        for (Map.Entry<String, Integer> pair : Storage.reports.entrySet()) {
```

Comment:
Using a static field from the Storage class to access reports ties this implementation to a specific storage mechanism. It would be better to inject a dependency on an interface that provides report data, which would make the class more flexible and testable.